### ISSUE TITLE
Audio conversion introduces distortions 

### ISSUE THREAD

**Issue by quamt** [rigaya/NVEnc#452]
Hello,

I am not sure if this is an issue on my side but I noticed since 7.08 that when the audio is converted it has distortions.
Either clipping or clicking. I tried to resolve the issue with the help of filters but it doesn't help.
And the source Audio doesn't seem to matter as it always happens sometimes stronger sometimes less audible but it is there. 

with filter
```
NVEncC (x64) 7.10 (r2404) by rigaya, Jan 19 2023 12:54:43 (VC 1929/Win)
OS Version     Windows 11 x64 (22621) [UTF-8]
CPU            AMD Ryzen 5 5600G with Radeon Graphics [4.21GHz] (6C/12T)
GPU            #0: NVIDIA GeForce GTX 1070 (1920 cores, 1771 MHz)[PCIe3x16][528.02]
NVENC / CUDA   NVENC API 12.0, CUDA 12.0, schedule mode: auto
Input Buffers  CUDA, 17 frames
Input Info     avsw: h264(yv12)->nv12 [AVX2], 1920x1080, 24000/1001 fps
Trim           0-1000 [offset: 0]
AVSync         vfr
Vpp Filters    copyHtoD
               cspconv(nv12 -> p010)
Output Info    H.265/HEVC main10 @ Level auto
               1920x1080p 1:1 23.976fps (24000/1001fps)
               avwriter: hevc,
                #1:aac/stereo:equalizer+dynaudnorm+aresample+ebur128 -> aac/stereo/192kbps
                => matroska
Encoder Preset quality
Rate Control   CQP  I:20  P:23  B:25
ChromaQPOffset cb:0  cr:0
Lookahead      off
GOP length     24 frames
B frames       0 frames [ref mode: disabled]
Ref frames     3 frames
AQ             on
CU max / min   auto / auto
VUI            chromaloc:left
Others         mv:auto aud pic-struct buf-period
```

without filter
```
NVEncC (x64) 7.10 (r2404) by rigaya, Jan 19 2023 12:54:43 (VC 1929/Win)
OS Version     Windows 11 x64 (22621) [UTF-8]
CPU            AMD Ryzen 5 5600G with Radeon Graphics [4.45GHz] (6C/12T)
GPU            #0: NVIDIA GeForce GTX 1070 (1920 cores, 1771 MHz)[PCIe3x16][528.02]
NVENC / CUDA   NVENC API 12.0, CUDA 12.0, schedule mode: auto
Input Buffers  CUDA, 17 frames
Input Info     avsw: h264(yv12)->nv12 [AVX2], 1920x1080, 24000/1001 fps
Trim           0-1000 [offset: 0]
AVSync         vfr
Vpp Filters    copyHtoD
               cspconv(nv12 -> p010)
Output Info    H.265/HEVC main10 @ Level auto
               1920x1080p 1:1 23.976fps (24000/1001fps)
               avwriter: hevc, #1:aac/stereo -> aac/stereo/192kbps => matroska
Encoder Preset quality
Rate Control   CQP  I:20  P:23  B:25
ChromaQPOffset cb:0  cr:0
Lookahead      off
GOP length     24 frames
B frames       0 frames [ref mode: disabled]
Ref frames     3 frames
AQ             on
CU max / min   auto / auto
VUI            chromaloc:left
Others         mv:auto aud pic-struct buf-period
```

**Comment by quamt** [comment]
Strangely enough it also happens with version 6.11.
I will try and rollback the NVIDIA driver.

**Comment by rigaya** [comment]
> And the source Audio doesn't seem to matter 

You mean this always happens no matter with or without filters?

I've run ```--audio-codec aac``` with some files, but haven't got this problem yet.

**Comment by quamt** [comment]
I tested video with AAC, AC3, DTS etc, whenever I convert Audio to AAC I get the problem.
I rolled back with the driver and it still persist. 
For verification purposes I tested on another machine with RTX 3080 vs this GTX 1070 and everything is ok.
I am not sure if the 1070 is defective.
I now try to update all component drivers to latest version and see if that helps.

**Comment by quamt** [comment]
I installed all the latest drivers and used dsim to try fix windows errors.
I not sure where else this problem can come from.
Playing the original audio it is ok. Converted is not ok.
Ill try another converter for the audio and see if the problem persists since its with this machine only it seems.

**Comment by quamt** [comment]
I tested the amd rx 6700 xt in the same machine and it has the same problem.
When using ffmpeg directly to encode the audio there is no distortion. It is very strange.

**Comment by quamt** [comment]
Ok. I am closing the issue for now. 
This seems to be related to the Mainboard MAG X570S TOMAHAWK MAX WIFI.
After doing some Google search.

**Comment by quamt** [comment]
In case anyone with the same mainboard comes across this issue here is how I solved it.

1. Update the USB audio firmware from MSI website 
2. Backup all your files and programs. Recommendation to download and save all drivers to separate HDD.
3. Update BIOS. Mine came with v1.41 however on MSI webpage there was only a 1.40 with a later date. So I updated to that version.
4. After that I reinstalled windows 11. A previous reset and reinstall without the BIOS update did not solve the isse. Windows had to be reinstalled cause SecureBoot was not enabled on previous BIOS.
5. Install all drivers and run a test.
